Packs/Gatewatcher/Integrations/GCenterv102/GCenterv102.yml

```diff
@@ -0,0 +1,1119 @@
+category: Network Security
+commonfields:
+  id: GCenterv102
+  version: -1
+configuration:
+- display: GCenter IP address
+  name: ip
+  type: 0
+  required: true
+- display: GCenter API token
+  name: token
+  type: 4
+  required: false
+- display: GCenter username
+  displaypassword: GCenter password
+  name: credentials
+  type: 9
```

Comment:
Maybe worth adding an 'additionalinfo:' key explaining that the user needs to provide either an API token or a username + password.